### PR TITLE
Entity updates (nest.com -> Google) and removals (imedia, gleam.io)

### PR DESCRIPTION
**Pull request by patjack**

### CODE REVIEW

File: disconnect-entitylist.json [hunk 1]
```diff
@@ -5337,6 +5338,7 @@
       "google.vu",
       "google.ws",
       "ingress.com",
+      "nest.com",
```
Comment (groovecoder) [inline disconnect-entitylist.json]: This will trigger a big entity list update again.

Comment (stpeter) [reply disconnect-entitylist.json]: Then can we split this PR into smaller chunks? IMHO it'd be great to process the removals separately.

Comment (patjack) [reply disconnect-entitylist.json]: No problem. I just created a new PR that omits the google change: https://github.com/mozilla-services/shavar-prod-lists/pull/85.

**Comment by englehardt** [comment]
@patjack would you mind to update this PR to only include the `nest.com` change? The others have already landed in #85. I'd still like to keep this change separate from #91 since it is blocked on #82.

**Comment by englehardt** [comment]
@patjack we recently diffed our master branch lists with yours to make sure we're still fully in sync and it looks like the `nest.com` change from this PR is the only remaining difference. Would you mind to update this PR (or submit a new one) that only includes the nest change? Thanks!

**Comment by patjack** [comment]
@englehardt no problem! I'll do that now.

**Comment by englehardt** [comment]
Closing in favor of #134. With that, all of these changes have been added in other PRs.

**Pull request closed without merging**